### PR TITLE
KingdomOverviewFragment with Logout button as the main screen

### PR DESCRIPTION
**Pull request by mozgaanna**
FragmentManager replaced with FragmentTransaction,
app.Fragment import replaced with support.v4.app.Fragment.

### CODE REVIEW

File: app/src/main/java/com/greenfox/peridot/peridot_coz_android/activity/MainActivity.java [hunk 1]
```diff
@@ -44,8 +55,22 @@ public void onResponse(Call<User> call, Response<User> response) {
...
+        drawer.setDrawerListener(toggle);
+        toggle.syncState();
+
+        NavigationView navigationView = (NavigationView) findViewById(R.id.nav_view);
+        navigationView.setNavigationItemSelectedListener(this);
+
+        KingdomOverviewFragment kingdomOverviewFragment= new KingdomOverviewFragment();
+        fragmentTransaction = getSupportFragmentManager().beginTransaction();
+        fragmentTransaction.replace(R.id.content_frame, kingdomOverviewFragment);
+        fragmentTransaction.commit();
```
Comment (adamgyulavari) [inline app/src/main/java/com/greenfox/peridot/peridot_coz_android/activity/MainActivity.java]: @radaladar `FragmentTransaction` is chainable, you don't need to store it in a variable, you can just call the methods on the previous ones

File: app/src/main/java/com/greenfox/peridot/peridot_coz_android/activity/MainActivity.java [hunk 2]
```diff
@@ -56,14 +81,88 @@ public void checkSharedPreferencesForUser(User user) {
         }
     }
 
-    public void logout(View v){
-        SharedPreferences preferences =getSharedPreferences("userInfo", Context.MODE_PRIVATE);
+    public void logout(){
+        SharedPreferences preferences = this.getSharedPreferences("userInfo", Context.MODE_PRIVATE);
```
Comment (adamgyulavari) [inline app/src/main/java/com/greenfox/peridot/peridot_coz_android/activity/MainActivity.java]: @radaladar `this` is not needed

File: app/src/main/java/com/greenfox/peridot/peridot_coz_android/activity/MainActivity.java [hunk 3]
```diff
@@ -56,14 +81,88 @@ public void checkSharedPreferencesForUser(User user) {
...
         SharedPreferences.Editor editor = preferences.edit();
         editor.putString("username", "");
         editor.putString("password", "");
         editor.apply();
         Toast.makeText(this,"Successful logout", Toast.LENGTH_SHORT).show();
-        Intent loginIntent = new Intent(MainActivity.this, LoginActivity.class);
-        startActivity(loginIntent);
+
+        Intent loginIntent = new Intent(this, LoginActivity.class);
+        this.startActivity(loginIntent);
```
Comment (adamgyulavari) [inline app/src/main/java/com/greenfox/peridot/peridot_coz_android/activity/MainActivity.java]: @radaladar `this` is not needed again, and also you can just instantiate the intent as the parameter

File: app/src/main/java/com/greenfox/peridot/peridot_coz_android/activity/MainActivity.java [hunk 4]
```diff
@@ -56,14 +81,88 @@ public void checkSharedPreferencesForUser(User user) {
...
+    @SuppressWarnings("StatementWithEmptyBody")
+    @Override
+    public boolean onNavigationItemSelected(MenuItem item) {
+        int id = item.getItemId();
+
+        if (id == R.id.nav_kingdom_overview) {
+            KingdomOverviewFragment kingdomOverviewFragment= new KingdomOverviewFragment();
+            fragmentTransaction = getSupportFragmentManager().beginTransaction();
+            fragmentTransaction.replace(R.id.content_frame, kingdomOverviewFragment);
+            fragmentTransaction.commit();
```
Comment (adamgyulavari) [inline app/src/main/java/com/greenfox/peridot/peridot_coz_android/activity/MainActivity.java]: @radaladar chaining here as well

**Review by adamgyulavari — CHANGES_REQUESTED** [review]

**Comment by radaladar** [comment]
@adamgyulavari requested changes done

**Review by adamgyulavari — APPROVED** [review]

**Pull request merged**